### PR TITLE
Supply appMenu close button label, for #1650

### LINKED ISSUE #1650: "Close" button of off-canvas navigation is not labelled

### PR DESCRIPTION
**Pull request by marcus-herrmann**
## Description
In unison with this PR in the design system (https://github.com/owncloud/owncloud-design-system/pull/497) with this PR it is possible to set the accessible name (`aria-label`) of the close button of the app sidebar menu

## Related Issue

- Fixes #1650 

## Motivation and Context
Before this PR the button presented itself as a nameless button to assistive technologies like screen readers. This PR suggest to name it "Close Menu".

## How Has This Been Tested?
*Work in Progress*

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [X] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [X] Tests

### CODE REVIEW

File: src/components/Menu.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <oc-application-menu name="coreMenu" v-model="sidebarIsVisible" @close="sidebarIsVisible = false">
+  <oc-application-menu name="coreMenu" v-model="sidebarIsVisible" @close="sidebarIsVisible = false" close-button-label="closeButtonLabel">
```
Comment (DeepDiver1975) [inline src/components/Menu.vue]: Is this working for you? Should be 
```vuejs
:close-button-label="closeButtonLabel"

Comment (marcus-herrmann) [reply src/components/Menu.vue]: It was, but still not the thing I originally intended. [Fixed](https://github.com/owncloud/phoenix/commit/1894bfc9b725d28bfab76ca9f72a94006882ec77).

**Comment by PVince81** [comment]
@marcus-herrmann is this ready for another review ? if it is, please ping people

**Comment by marcus-herrmann** [comment]
Ready for further review @LukasHirt @DeepDiver1975

File: src/components/Menu.vue [hunk 2]
```diff
@@ -20,7 +20,8 @@ export default {
   data () {
     return {
       isOpen: false,
-      appVersion: appVersionJson
+      appVersion: appVersionJson,
+      closeButtonLabel: this.$gettext('Close Menu')
```
Comment (LukasHirt) [inline src/components/Menu.vue]: Make this pls computed property or use it inline.

**Comment by marcus-herrmann** [comment]
@LukasHirt Done.

**Comment by PVince81** [comment]
conflict.

also will need to redo this after https://github.com/owncloud/phoenix/pull/2669 is merged as we'll have more menus to apply this to.

I can take care

**Comment by pascalwengerter** [comment]
Closing this since both the `oc-sidebar-nav`'s and the `files-sidebar`'s close buttons/icons have `aria-label`s now (see web.owncloud.com)

**Pull request closed without merging**